tools/clang/lib/SPIRV/SPIRVEmitter.cpp

```diff
@@ -4484,7 +4513,8 @@ uint32_t SPIRVEmitter::processIntrinsicFrexp(const CallExpr *callExpr) {
     }
   }
 
-  emitError("Unknown argument type passed to Frexp function.");
+  emitError("unknown argument type passed to Frexp function",
```

Comment:
invalid argument type passed to Frexp intrinsic function?

Reply:
Done